Functions.Templates/Templates/DurableFunctionsActivity-CSharp/build.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": {
+    "text": "Durable Functions Activity trigger",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "1002"
+  },
+  "description": {
+    "text": "A C# function that will be run whenever an Activity is trigger",
```

Comment:
>A C# function that will be run whenever an Activity is trigger [](start = 13, length = 62)

This text is weird from a grammar perspective. Should it maybe say "A C# function that will be run whenever an Activity is called by an Orchestrator function" or something along those lines?